docs/_ext/custom_styles/formatter.py

```diff
@@ -38,8 +38,7 @@ def format_header(self, lines: List[str]) -> List[str]:
     def format_overview(self, lines: List[str]) -> List[str]:
         """Format overview section."""
         format_lines = [
-            ""
-            ".. rubric:: Overview",
+            "" ".. rubric:: Overview",
```

Comment:
I think it is supposed to be

```suggestion
            "",
            ".. rubric:: Overview",
```